### DISCUSSION TITLE
Screen flickering while having another full screen GPU utilizing application open

### DISCUSSION THREAD

**Discussion by SleepingDrone98** [runelite/runelite#18370] [Q&A]
I had Runelite full screened overtop of WoW, doing so caused a very distracting flickering in OSRS.

Just curious if it's runelite related (I'm using the HD plugin)

**Comment by cdfisher** [comment]
Since you're using 117, the more relevant place to ask would be over on their repo [here](https://github.com/117HD/RLHD/issues).
